### PR TITLE
Medical guide improvements

### PR DESCRIPTION
**Pull request by gienkov**
Mainly based on my community experience with CMS, translated from exact same PL guide.

### CODE REVIEW

File: documentation/feature/medical-system.md [hunk 1]
```diff
@@ -56,7 +66,7 @@ In order to stop the bleeding, all injuries on every bodypart requires treatment
 While a unit is bleeding however, the blood volume decreases which will result in a change of vitals.  Depending on the factors such as current blood volume, the blood loss rate, medication used, the blood pressure will start to drop. To counter this drop, also based upon the previously mentioned factors and others, the heart rate will adjust accordingly to attempt to keep blood pressure at safe levels. This means that for any patient it is required to keep an eye on the vitals. This is done through the interaction system by selecting check pulse or blood pressure on either the arms or head.
 
 #### 3.1.3 Medication
-To stabalize the vitals and to counter for example pain, a player/medic can use medication. Advanced medical has 3 different medications available:
+To stabalize the vitals and to counter for example pain, a player/medic can use medication. All medications weights the same: 50 grams. Advanced medical has 3 different medications available:
```
Comment (Glowbal) [inline documentation/feature/medical-system.md]: I don't think weight of equipment is relevant to the medication chapter.

File: documentation/feature/medical-system.md [hunk 2]
```diff
@@ -68,3 +78,144 @@ Morphine is used to alleviate large amounts of pain. Has an effect similar to He
...
+  * Velocity wounds - high effectiveness
+  * Puncture wounds - high effectiveness
+* Tourniquet
+  * Can only be applied on limbs
+  * Stops bleeding from wounds
+  * Should be taken off as fast as possible and applied only to give medic time to bandage all the wounds
+  * If not taken off for a while it will cause pain to patient, can yield to death that way
+
+#### 3.1.6 Transfuzions
+In case of blood loss, blood should be replenished by three different types of IV: blood, plasma and saline. We distinguish 3 different sizes of IV bags: 250ml, 500ml and 1000ml.
```
Comment (Glowbal) [inline documentation/feature/medical-system.md]: should -> could

You can use any of the variants at the moment.

File: documentation/feature/medical-system.md [hunk 3]
```diff
@@ -68,3 +78,144 @@ Morphine is used to alleviate large amounts of pain. Has an effect similar to He
...
+Transfuzing 250ml of given IV takes about 1 minute to complete!
+
++#### 3.1.7 PAK and Surgical kit
+Using Personal Aid Kit brings patient to the best possible health state. Depending on module options, PAK can be used only in specified situations, specified place and by specified person.
+Surgical kit is used to stich bandaged wounds so they will never open again. Depending on module options, surgical kit can be used only in specified situations, specified place and by specified person.
+
+### 3.2 Advanced medical system - recommended gear
+ 
+* Soldier:
```
Comment (Glowbal) [inline documentation/feature/medical-system.md]: I'd recommend one morphine.

File: documentation/feature/medical-system.md [hunk 4]
```diff
@@ -68,3 +78,145 @@ Morphine is used to alleviate large amounts of pain. Has an effect similar to He
...
+  * Puncture wounds - lowest effectiveness
+* Bandage (elastic)
+  * Abrasions - highest effectiveness
+  * Avulsions - lowest effectiveness
+  * Contusions - highest effectiveness
+  * Crush wounds - highest effectiveness
+  * Cut wounds - highest effectiveness
+  * Lacerations - highest effectiveness
+  * Velocity wounds - low effectiveness
+  * Puncture wounds - wysoka effectiveness
```
Comment (Aatu) [inline documentation/feature/medical-system.md]: wysoka effectiveness <- missing translation

**Comment by gienkov** [comment]
All fixed

**Comment by Glowbal** [comment]
:+1: I will merge this once I am on a desktop again.

**Comment by gienkov** [comment]
Thanks! I will continue on improving it.
15 maj 2015 15:10 "Glowbal" <notifications@github.com> napisał(a):

**Comment by Glowbal** [comment]
Merged, thank you!

**Pull request merged**